tests/e2e/workloads/bdi/helper.py

```diff
@@ -0,0 +1,143 @@
+"""
```

Comment:
we have recently transferred all helper modules to be under ocs_ci/helpers, please move this file there

Reply:
These helpers are specific for BDI workload.
Should I move them anyway?